2.7/ubuntu18.04/Dockerfile

```diff
@@ -0,0 +1,100 @@
+FROM ubuntu:18.04@sha256:b88f8848e9a1a4e4558ba7cfc4acc5879e1d0e7ac06401409062ad2627e6fb58
```

Comment:
ah, have we been locking it down to shas now?

Reply:
Guess so?